src/mtool/ui/src/components/VolumeManagement/AdvanceCreateVolume/index.js

```diff
@@ -978,15 +997,15 @@ function AdvanceCreateVolume(props) {
       aria-describedby="modal-modal-description"
     >
       <Box sx={style}>
-        <IconButton className={classes.cancleIcon} aria-label="Close" onClick={handleModalClose}>
-          <Cancel />
-        </IconButton>
+        <DialogTitle onClose={handleModalClose}>
+          Advance Create Volume
+        </DialogTitle>
         <form className={classes.formContainer}>
           <main className={classes.main}>
             <div className={classes.mainContent}>
               <div>
-                <Typography className={classes.createHeader}>Create Volume</Typography>
-                <Stepper activeStep={activeStep} className={classes.xsHide} orientation="vertical">
+                <Typography className={classes.createHeader}>Steps</Typography>
+                <Stepper activeStep={activeStep} className={classes.xsHide} style={{ padding: "16px" }}orientation="vertical">
```

Comment:
Is it possible to move the styling inside a class?

Reply:
yes I have added a new class